### PR TITLE
Jetty-9 sign commit and tag with release script

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

### CODE REVIEW

**Comment by joakime** [comment]
I'm not convinced this is the right place to do this.

I have multiple gpg keys, and your changes will pick the first one (which isn't the one to use)

Perhaps we should document the various `$HOME/.gitconfig` options instead?

Like ...

```
[user]
      signingkey = ABCEV

[commit]
      gpgsign = true
```

etc... ?

File: scripts/release-jetty.sh [hunk 1]
```diff
@@ -156,12 +156,12 @@ if proceedyn "Are you sure you want to release using above? (y/N)" n; then
             -DprocessAllModules=true 
     fi
     if proceedyn "Commit $VER_RELEASE updates? (Y/n)" y; then
-        git commit -a -m "Updating to version $VER_RELEASE"
+        git commit -sa -m "Updating to version $VER_RELEASE"
```
Comment (joakime) [inline scripts/release-jetty.sh:159]: This is handled by `git config commit.gpgsign=true` instead.

File: scripts/release-jetty.sh [hunk 2]
```diff
@@ -156,12 +156,12 @@ if proceedyn "Are you sure you want to release using above? (y/N)" n; then
...
     fi
     if proceedyn "Create Tag $TAG_NAME? (Y/n)" y; then
         echo "TODO: Sign tags with GIT_USER_EMAIL=$GIT_USER_EMAIL"
         echo "Using target/version-tag.txt as tag text"
-        git tag --file=target/version-tag.txt $TAG_NAME
+        git -s tag --file=target/version-tag.txt $TAG_NAME
```
Comment (joakime) [inline scripts/release-jetty.sh:164]: This is handled by `git config tag.gpgsign=true` instead.

File: scripts/release-jetty.sh [hunk 3]
```diff
@@ -181,7 +181,7 @@ if proceedyn "Are you sure you want to release using above? (y/N)" n; then
             -DprocessAllModules=true 
         echo "Commit $VER_NEXT"
         if proceedyn "Commit updates in working directory for $VER_NEXT? (Y/n)" y; then
-            git commit -a -m "Updating to version $VER_NEXT"
+            git commit -sa -m "Updating to version $VER_NEXT"
```
Comment (joakime) [inline scripts/release-jetty.sh:184]: This is handled by `git config commit.gpgsign=true` instead.

**Review by joakime — CHANGES_REQUESTED** [review]
Use these command lines instead ...

```
git config --global commit.gpgSign "true"
git config --global tag.gpgSign "true"
```

And if you have multiple keys to sign, set the right one with ...

```
git config --global user.signingKey <gpg-keyid>
```

**Comment by olamy** [comment]
I do have this setup configured locally but if someone doesn't have it this generates non-signed tags as we have here https://github.com/eclipse/jetty.project/tags
should check it in the script?

**Comment by olamy** [comment]
should be fixed with correct user setup

**Pull request closed without merging**